docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md

```diff
@@ -62,18 +62,22 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
 |`Switch.MS.Internal.`<br/>`UseSha1AsDefaultHashAlgorithmForDigitalSignatures`|Controls whether the default algorithm used for signing parts of a package by PackageDigitalSignatureManager is SHA1 or SHA256.|.NET Framework 4.7.1|
```

Comment:
vendor task to change this table to maybe a list with H3 headers? It looks really awful.